docs/rfcs/0000-namespace.md

```diff
@@ -0,0 +1,115 @@
+we use `active_model`.
+
+And based on the [bump of `0.10.0.pre` released by Steve Klabnik](https://github.com/rails-api/active_model_serializers/tree/86fc7d7227f3ce538fcb28c1e8c7069ce311f0e1)
+if we take a look on README and gemspec always is used `ActiveModel::Serializers`.
+
+## New classes and modules organization
+
+Since this will be a big change we can do this on baby steps, read small PRs. A
+possible approach is:
+
+- Create the `ActiveModel::Serializers` namespace;
+- Move all content under `ActiveModelSerializers` to be under
+  `ActiveModel::Serializers`, the logger is on this step;
+- Move all content under `ActiveModel::Serializer` to be under
+  `ActiveModel::Serializers`, the adapter is on this steps;
+- Move all content under `ActiveModel` to be under `ActiveModel::Serializers`,
+  the `SerializableResource` is on this step;
+- Now that all the code lives under `ActiveModel::Serializers` we can:
+  - create a better name to the `ActiveModel::Serializers::Serializer`
+    keeping in mind only to keep this in the same namespace
+  - create a better name to the `ActiveModel::Serializers::Serializer::Adapter::JsonApi`
+    probably remove this from the `ActiveModel::Serializers::Serializer`
+    and do something like `ActiveModel::Serializers::Adapter::JsonApi`
+    keeping in mind only to keep this in the same namespace
+  - Change all public API that doesn't make sense, keeping in mind only to keep
+    this in the same namespace
+- Update the README;
+- Update the docs;
+
+The following table represents the current and the desired classes and modules
+at the first moment.
+
+| Current                                               | Desired                                          | Notes              |
+|-------------------------------------------------------|--------------------------------------------------|--------------------|
+|`ActiveModelSerializers` and `ActiveModel::Serializer` | `ActiveModel::Serializers`                       | The main namespace |
+| `ActiveModelSerializers.logger`                       | `ActiveModel::Serializers.logger`                ||
+|`ActiveModelSerializers::Model`                        | `ActiveModel::Serializers::Model`                ||
+|`ActiveModel::SerializableResource`                    | `ActiveModel::Serializers::SerializableResource` ||
+| `ActiveModel::Serializer`                             | `ActiveModel::Serializers::Serializer`           | I know that is probably a bad name, but In a second moment we can rename this to `Resource` [for example following this idea](https://github.com/rails-api/active_model_serializers/pull/1301/files#r42963185)|
+|`ActiveModel::Serializer.config`                       | `ActiveModel::Serializers.config`                ||
+
+# Drawbacks
+
+This will be a breaking change, so all users serializers will be broken.
+All PRs will need to rebase since the architeture will change a lot.
```

Comment:
My preference would be to add aliases from the old namespaces to the new ones, and just throw some deprecation warnings.

Using this, http://apidock.com/ruby/Module/const_missing
it would be pretty easy to log a deprecation message, and then proceed to the re-assigned namespace.